### PR TITLE
Add few clarifications to our security policy

### PR DESCRIPTION
**Pull request by potiuk**
* Brought clearer instructions on how to report issues with one issue per email in plain text from the https://www.apache.org/security/

* Provided specific exclusions for security scan results for image, adding instructions on how to build your own image and how Airflow image is produced

* Brought information from https://www.apache.org/security/ that **only** undisclosed security issues can be handled via security@a.a.o not general dicussions on security and bug reports

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: .github/SECURITY.md [hunk 1]
```diff
@@ -49,7 +60,11 @@ in other products/circumstances. Therefore, some potential security vulnerabilit
 apply to Airflow, or have a different severity than some generic scoring systems (for example `CVSS`)
 calculation suggests.
 
-The [ASF Security team's page](https://www.apache.org/security/) describes
+The security team (see below "Handling security issues in Airflow") will get back to
```
Comment (pankajkoti) [inline .github/SECURITY.md]:
```suggestion
The security team (see below [Handling security issues in Airflow](https://github.com/apache/airflow/security#handling-security-issues-in-airflow)) will get back to
```

Comment (pankajkoti) [reply .github/SECURITY.md]: perhaps this may work too? (if we would like to consider :) )

File: .github/SECURITY.md [hunk 2]
```diff
@@ -31,14 +31,25 @@ Airflow specifically offers security features and is responsive to issues
...
+Specifically, we will ignore results of security scans that contain list of
+dependencies of Airflow with dependencies in Airflow Docker reference image - there
+is a page that describes how the
+[Airflow reference Image is fixed at release time](https://airflow.apache.org/docs/docker-stack/index.html#fixing-images-at-release-time)
+and providing helpful instructions explaining general instruction on how you can build your
+own image and manage dependencies of Airflow in your own image.
+
+**Please send one plain-text email for each vulnerability you are reporting including explanation
+how it affects airflow security. We may ask you to resubmit your report if you send it as an image,
+~~movie, HTML, or PDF attachment  when you could as easily describe it with plain text.**~~
```
Comment (pankajkoti) [inline .github/SECURITY.md]:
```suggestion
movie, HTML, or PDF attachment  when you could as easily describe it with plain text.**
```

Suggesting to remove the ~~, but unsure if it was put their intentionally and what it means for Markdown.

**Review by pankajkoti — APPROVED** [review]

File: .github/SECURITY.md [hunk 3]
```diff
@@ -31,14 +31,25 @@ Airflow specifically offers security features and is responsive to issues
...
+sent to this address that does not relate to an undisclosed security problem
+in the Apache Airflow project. Please follow regular communication channels described in
+the [Airflow Community](https://airflow.apache.org/community/index.html) page for
+inquiries, questions and other discussions related to the process or issues.
+
+Specifically, we will ignore results of security scans that contain list of
+dependencies of Airflow with dependencies in Airflow Docker reference image - there
+is a page that describes how the
+[Airflow reference Image is fixed at release time](https://airflow.apache.org/docs/docker-stack/index.html#fixing-images-at-release-time)
+and providing helpful instructions explaining general instruction on how you can build your
```
Comment (phanikumv) [inline .github/SECURITY.md]:
```suggestion
and providing helpful instructions explaining how you can build your
```

**Review by phanikumv — APPROVED** [review]

**Comment by potiuk** [comment]
Any more comments?

**Review by pierrejeambrun — APPROVED** [review]

File: .github/SECURITY.md [hunk 4]
```diff
@@ -31,14 +31,25 @@ Airflow specifically offers security features and is responsive to issues
...
-before disclosing it publicly.
+**Only** use the security contacts to report undisclosed security vulnerabilities in Apache
+Airflow and manage the process of fixing such vulnerabilities. We cannot accept regular
+bug reports or other security-related queries at these addresses. We will ignore mail
+sent to this address that does not relate to an undisclosed security problem
+in the Apache Airflow project. Please follow regular communication channels described in
+the [Airflow Community](https://airflow.apache.org/community/index.html) page for
+inquiries, questions and other discussions related to the process or issues.
+
+Specifically, we will ignore results of security scans that contain list of
```
Comment (o-nikolas) [inline .github/SECURITY.md]:
```suggestion
Specifically, we will ignore results of security scans that contain a list of
```

File: .github/SECURITY.md [hunk 5]
```diff
@@ -31,14 +31,25 @@ Airflow specifically offers security features and is responsive to issues
...
+dependencies of Airflow with dependencies in Airflow Docker reference image - there
+is a page that describes how the
+[Airflow reference Image is fixed at release time](https://airflow.apache.org/docs/docker-stack/index.html#fixing-images-at-release-time)
+and providing helpful instructions explaining how you can build your
+own image and manage dependencies of Airflow in your own image.
+
+**Please send one plain-text email for each vulnerability you are reporting including explanation
+how it affects airflow security. We may ask you to resubmit your report if you send it as an image,
```
Comment (o-nikolas) [inline .github/SECURITY.md]:
```suggestion
**Please send one plain-text email for each vulnerability you are reporting including an explanation
of how it affects Airflow security. We may ask that you resubmit your report if you send it as an image,
```

File: .github/SECURITY.md [hunk 6]
```diff
@@ -31,14 +31,25 @@ Airflow specifically offers security features and is responsive to issues
...
+movie, HTML, or PDF attachment  when you could as easily describe it with plain text.**
```
Comment (o-nikolas) [inline .github/SECURITY.md]:
```suggestion
movie, HTML, or PDF attachment when you could as easily describe it with plain text.**
```

File: .github/SECURITY.md [hunk 7]
```diff
@@ -31,14 +31,25 @@ Airflow specifically offers security features and is responsive to issues
...
+inquiries, questions and other discussions related to the process or issues.
+
+Specifically, we will ignore results of security scans that contain list of
+dependencies of Airflow with dependencies in Airflow Docker reference image - there
+is a page that describes how the
+[Airflow reference Image is fixed at release time](https://airflow.apache.org/docs/docker-stack/index.html#fixing-images-at-release-time)
+and providing helpful instructions explaining how you can build your
+own image and manage dependencies of Airflow in your own image.
+
+**Please send one plain-text email for each vulnerability you are reporting including explanation
```
Comment (o-nikolas) [inline .github/SECURITY.md]: What's the background on why we demand this? I have found it very helpful when images or detailed documents have been included with the report.

File: .github/SECURITY.md (hunk 6 above)
Comment (jedcunningham) [inline .github/SECURITY.md]:
```suggestion
Please send one plain-text email for each vulnerability you are reporting including explanation
how it affects airflow security. We may ask you to resubmit your report if you send it as an image,
movie, HTML, or PDF attachment  when you could as easily describe it with plain text.
```

I don't think we should bold this whole paragraph.

File: .github/SECURITY.md [hunk 8]
```diff
@@ -49,7 +60,11 @@ in other products/circumstances. Therefore, some potential security vulnerabilit
 apply to Airflow, or have a different severity than some generic scoring systems (for example `CVSS`)
 calculation suggests.
 
-The [ASF Security team's page](https://www.apache.org/security/) describes
+The security team (see below [Handling security issues in Airflow](https://github.com/apache/airflow/security#handling-security-issues-in-airflow)) will get back to
+you after assessing the report.
```
Comment (o-nikolas) [inline .github/SECURITY.md]:
```suggestion
The security team will get back to you after assessing the report (see below [Handling security issues in Airflow](https://github.com/apache/airflow/security#handling-security-issues-in-airflow)).
```

File: .github/SECURITY.md [hunk 9]
```diff
@@ -31,14 +31,25 @@ Airflow specifically offers security features and is responsive to issues
...
 
-Note that this security address should be used only for undisclosed
-vulnerabilities. Dealing with fixed issues or general questions on how to use
-the security features should be handled regularly via the user and the dev
-lists. Please report any security problems to the project security address
-before disclosing it publicly.
+**Only** use the security contacts to report undisclosed security vulnerabilities in Apache
+Airflow and manage the process of fixing such vulnerabilities. We cannot accept regular
+bug reports or other security-related queries at these addresses. We will ignore mail
+sent to this address that does not relate to an undisclosed security problem
```
Comment (o-nikolas) [inline .github/SECURITY.md]:
```suggestion
**Only** use the security e-mail address to report undisclosed security vulnerabilities in Apache
Airflow and to manage the process of fixing such vulnerabilities. We do not accept regular
bug reports or other security-related queries at this address. We will ignore mail
sent to this address that does not relate to an undisclosed security problem
```

File: .github/SECURITY.md (hunk 7 above)
Comment (potiuk) [reply .github/SECURITY.md]: This is straight copy from the ASF rules. While useful to have them (and we do not reject them if accompanied by text description), having videos or images is not something that is easy copy-pasteable and reproducible. For example if you have a POST request with some details, it's rather hard to try to reproduce it based on the video, if you need to copy&paste the post.

File: .github/SECURITY.md [hunk 10]
```diff
@@ -49,7 +60,10 @@ in other products/circumstances. Therefore, some potential security vulnerabilit
 apply to Airflow, or have a different severity than some generic scoring systems (for example `CVSS`)
 calculation suggests.
 
-The [ASF Security team's page](https://www.apache.org/security/) describes
+The security team will get back to you after assessing the report (see below [Handling security issues in Airflow](https://github.com/apache/airflow/security#handling-security-issues-in-airflow)).
```
Comment (jedcunningham) [inline .github/SECURITY.md]:
```suggestion
The [Airflow Security Team](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#security-team) will get back to you after assessing the report.
```

After #32496, let's do this instead.

File: .github/SECURITY.md [hunk 11]
```diff
@@ -31,14 +31,25 @@ Airflow specifically offers security features and is responsive to issues
...
+Specifically, we will ignore results of security scans that contain a list of
+dependencies of Airflow with dependencies in Airflow Docker reference image - there
+is a page that describes how the
+[Airflow reference Image is fixed at release time](https://airflow.apache.org/docs/docker-stack/index.html#fixing-images-at-release-time)
+and providing helpful instructions explaining how you can build your
+own image and manage dependencies of Airflow in your own image.
+
+**Please send one plain-text email for each vulnerability you are reporting including an explanation
+of how it affects Airflow security. We may ask that you resubmit your report if you send it as an image,
+movie, HTML, or PDF attachment when you could as easily describe it with plain text.**
```
Comment (potiuk) [inline .github/SECURITY.md]:
```suggestion
Please send one plain-text email for each vulnerability you are reporting including an explanation
of how it affects Airflow security. We may ask that you resubmit your report if you send it as an image,
movie, HTML, or PDF attachment when you could as easily describe it with plain text.
```

File: .github/SECURITY.md (hunk 6 above)
Comment (potiuk) [reply .github/SECURITY.md]: yea. Let's see how many of those "10 security issues from the scanner" we get :). But yes - no problem with not-bolding.

**Review by jedcunningham — APPROVED** [review]

**Pull request merged**